### ISSUE TITLE
Support for 'Going Live' and 'Shipped' command

### ISSUE THREAD

**Issue by mattstibbs** [mattstibbs/slack_changebot#7]
It would be good if you could trigger a 'going live' status for a change with a simple command.

For example you could type `/111 going live`, and this could:

- check if you're in a change channel
- if so, post a message to the change channel saying "<User> has notified this change is now going live!"
- also post a message to the #announcements channel saying "Change <no> is now going live!"

Once a change has succesfully gone live, a similar `/111 shipped` command could follow suit

**Comment by ShaneHudson** [comment]
Yeah I like this idea, it could probably find and share the release notes which is nice. Especially for when people forget to pin them.

**Comment by mattstibbs** [comment]
Suggested implementation:

`/111 staging` - Announce that change is going to staging
`/111 going live` - Announce that change is shipping to live
`/111 shipped` - Announce that the change is now live (shipped)

Both options would put a message into both the change channel, and the #announcements (configurable) channel.

The commands should only work when executed from within a change channel, otherwise it should display an ephemeral message back to the user saying that they can only be used within an change channel.
